greenpithumb/poller.py

```diff
@@ -190,3 +190,23 @@ def _poll_once(self):
             if ml_pumped > 0:
                 self._watering_event_store.store_water_pumped(
                     self._local_clock.now(), ml_pumped)
+
+
+class CameraPoller(SensorPollerBase):
```

Comment:
Oh this is clever! It hadn't occurred to me that it would just plug into our pollers.